src/scss/_objects.resources.scss

```diff
@@ -0,0 +1,16 @@
+.resources {
+
+  &__subheader.paragraph {
+    margin-top: 0;
```

Comment:
If you wrap the `Heading` and `.resources__subheader` in a `div`, I believe you shouldn't need this style.